package.json

```diff
@@ -25,6 +25,7 @@
     "@bugsnag/js": "^7.11.0",
     "@bugsnag/plugin-aws-lambda": "^7.11.0",
     "@conveyal/lonlat": "^1.4.1",
+    "@opentripplanner/geocoder": "^1.1.2",
```

Comment:
I would throw a blocked tag on this PR until the new geocoder update is published and updated here.

Reply:
It is ready!